yarn-project/bot/src/bot.ts

```diff
@@ -28,7 +28,7 @@ export class Bot {
     public readonly config: BotConfig,
   ) {}
 
-  static async create(config: BotConfig, dependencies: { pxe?: PXE } = {}): Promise<Bot> {
+  static async create(config: BotConfig, dependencies: { pxe?: PXE; node?: AztecNode } = {}): Promise<Bot> {
```

Comment:
don't see the new `node` arg being used anywhere?

Reply:
It's just through, into the BotFactory constructor.

Reply:
Ah right it's part of the dependencies obj, got it 👍